tasks/helpers/release-folder-helper.js

```diff
@@ -2,15 +2,10 @@ import {map, pipeline} from 'event-stream';
 import gulp from 'gulp';
 import path from 'path';
 
-import {getNewVersion} from './version-helper';
-
 export function releaseDest(folderName='') {
   const prefixReleaseToDestStream = map(async (file, callback) => {
     try {
-      const version = getNewVersion();
-      if (file.path.indexOf(`pui-v${version}`) === -1) {
-        file.path = path.join(file.base, `pui-v${version}`, folderName, file.relative);
-      }
+      if (folderName) file.path = path.join(file.base, folderName, file.relative);
```

Comment:
@rockbot We could potentially keep it, but my argument is that a version-specific directory becomes unnecessary when using tags. E.g. if we want to reference release artifacts via GitHub url, I'd rather have a url like `https://raw.githubusercontent.com/npm/pivotal-ui/v5.1.2/release/pivotal-ui.min.css` instead of `https://raw.githubusercontent.com/npm/pivotal-ui/v5.1.2/release/pui-v5.1.2/pivotal-ui.min.css` (first version in url is the tag name, second version is superfluous).